tests/test_smoke.py

```diff
@@ -1,16 +1,2 @@
-import pytest
-    
-
-def test_error_without_env_var() -> None:
```

Comment:
just curious: why did you remove this test?

Reply:
with the environment variable set via Git Actions, it would be cumbersome to have this variable deleted system-wide and then re-defined to be used on other tests.